fiovb/Makefile

```diff
@@ -3,6 +3,12 @@ export V ?= 0
 HOST_CROSS_COMPILE ?= $(CROSS_COMPILE)
 TA_CROSS_COMPILE ?= $(CROSS_COMPILE)
 
+ifdef CFG_FIOVB_VENDOR_CREATE
+ifndef CFG_FIOVB_VENDOR_PREFIX
+$(error CFG_FIOVB_VENDOR_CREATE is set but no CFG_FIOVB_VENDOR_PREFIX was provided)
+endif
+endif
+
 .PHONY: all
```

Comment:
The same can be done on the CMakeLists.txt and is better at configure stage than when compiling
```
if(DEFINED CFG_FIOVB_VENDOR_CREATE AND NOT DEFINED)
  message( FATAL_ERROR "CFG_FIOVB_VENDOR_CREATE is set but no CFG_FIOVB_VENDOR_PREFIX was provided." )
endif()
```
I haven't tested it but I think this will work

Reply:
right, I have checked and the recipe [optee-fiovb_git.bb](https://github.com/foundriesio/meta-lmp/blob/main/meta-lmp-base/recipes-security/optee/optee-fiovb_git.bb) is using the `oe_runmake` directly .